pkgs/development/tools/pgFormatter/default.nix

```diff
@@ -0,0 +1,39 @@
+{ buildPerlPackage, fetchFromGitHub, glibcLocales, lib, ... }:
+
+# original implementation stolen from https://github.com/ariutta/nixpkgs-custom/blob/aa72a115a71bb248a2a7f0deac55132e68b54365/perl-packages.nix#L13
+
+buildPerlPackage rec {
+  name = "pgFormatter-${version}";
+  version = "v3.2";
```

Comment:
I don't think the version should have a `v` in it.
(since you're using this in the `name` too)

So then the `rev` in `fetchFromGitHub` should be 
```
rev    = "v${version}";
```

Reply:
ok, will fix